Standards/scs-XXXX-vN-taxonomy-of-failsafe-levels.md

```diff
@@ -0,0 +1,264 @@
+TODO: What time frame do we look at? (so called Recovery Time Objecte aka RTO)
+TODO: how does this relate to Business Continuity Planning (BCP)
+-->
+
+## Glossary
+
+| Term               | Explanation                                                                                                                              |
+| ------------------ | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| Availability Zone  | (also: AZ) internal representation of physical grouping of service hosts, which also lead to internal grouping of resources.             |
+| BSI                | German Federal Office for Information Security (Bundesamt für Sicherheit in der Informationstechnik).                                    |
+| CSP                | Cloud Service Provider, provider managing the OpenStack infrastructure.                                                                  |
+| Compute            | A generic name for the IaaS service, that manages virtual machines (e.g. Nova in OpenStack).                                             |
+| Network            | A generic name for the IaaS service, that manages network resources (e.g. Neutron in OpenStack).                                         |
+| Storage            | A generic name for the IaaS service, that manages the storage backends and virtual devices (e.g. Cinder in OpenStack).                   |
+
+## Context
+
+Some standards provided by the SCS project will talk about or require procedures to back up resources or have redundancy for resources.
+This decision record should discuss, which failure threats are CSP-facing and will classify them into several levels.
+In consequence these levels should be used in standards concerning redundancy or failure safety.
+
+Based on our research, no similar standardized classification scheme seems to exist currently.
+Something close but also very detailed is the [BSI-Standard 200-3 (german)][bsi-200-3] published by the German Federal Office for Information Security.
+As we want to focus on IaaS and K8s resources and also have an easily understandable structure that can be applied in standards covering replication, redundancy and backups, this document is too detailed.
+
+[bsi-200-3]: https://www.bsi.bund.de/SharedDocs/Downloads/DE/BSI/Grundschutz/BSI_Standards/standard_200_3.pdf?__blob=publicationFile&v=2
+
+## Decision
+
+### Failsafe Levels
+
+This Decision Record defines **four** failsafe levels, each of which describe what kind of failures have to
+be tolerated by a provided service.
+
+In general, the lowest, **level 1**, describes isolated/local failures which can occur very frequently, whereas
+the highest, **level 4**, describes relatively unlikely failures that impact a whole or even multiple datacenter(s):
+
+| Level | Probability | Impact                | Examples |
```

Comment:
I'm still not sure, if we should start with this. I know there are two was to present a decision. Mine was: 1. explain everything that leads to that decision and 2. the decision.
Now it is the other way around: 1. decision 2. why did we come to that decision.

This is just my personal style: I like having the explanation before. But we can have a discussion about this, maybe for this long long topic, people do not want to read through everything and come to the most important table quickly.